web/index.html

```diff
@@ -376,32 +437,36 @@ <h3>Used in</h3>
             return queryParams;
         }
 
-        // Decode base64 value mod 1000
-        function getChunk(value) {
```

Comment:
Moved this function to be a method instead since it is now used directly in the hover events -- tried to make it a computed function, but for some reason that was giving me errors